### PR TITLE
Fabric project submodule initial import

### PR DESCRIPTION
**Pull request by ludoo**
As agreed, this is the initial import of our project module, as a submodule.

### CODE REVIEW

File: modules/fabric-project/variables.tf [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "parent_id" {
+  description = "Id of the resource under which the folder will be placed."
+}
+
+variable "parent_type" {
+  description = "Type of the parent reosurce, defaults to organization."
```
Comment (morgante) [inline modules/fabric-project/variables.tf]: Spelling.

File: modules/fabric-project/variables.tf [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+variable "prefix" {
+  description = "Prefix used to generate project id and name"
+}
+
+variable "name" {
+  description = "Project name and id suffix."
+}
+
+variable "billing_account_id" {
```
Comment (morgante) [inline modules/fabric-project/variables.tf]: Let's standardize on the same name for shared variables with the standard module:
- billing_account
- activate_apis

Comment (ludoo) [reply modules/fabric-project/variables.tf]: I'm not too keen on the `activate_apis` name, as it masks the actual resource name. But can see the benefits of standardizing, lemme fix that.

**Comment by morgante** [comment]
@ludoo LGTM, merged.

In a future update, we should consider whether we want to offer an easy path for moving between different versions of the module.

**Pull request merged**